### ISSUE TITLE
Version conflict detected when installing EF 2.1.1 on ASP Core 2.1

### ISSUE THREAD

**Issue by gnpretorius** [dotnet/efcore#12408]
**Describe what is not working as expected.**
When adding a reference to Microsoft.EntityFrameworkCore 2.1.1 on a brand new ASP Core 2.1 Web API, I get the following error:

 ```
Version conflict detected for Microsoft.EntityFrameworkCore.Abstractions. Reference the package directly from the project to resolve this issue. 

Sample -> Microsoft.EntityFrameworkCore 2.1.1 -> Microsoft.EntityFrameworkCore.Abstractions (>= 2.1.1)
 
Sample -> Microsoft.AspNetCore.App 2.1.0 -> Microsoft.EntityFrameworkCore.Abstractions (= 2.1.0).
```

### Steps to reproduce
1. Create a new Website using Asp Core 2.1
2. Select WebApi 
3. Manage Nuget packages
4. Add a reference to Microsoft.EntityFrameworkCore 2.1.1
5. Error...

### Further technical details
EF Core version: 2.1.1
Database Provider: NA
Operating system: Windows 10 Pro
IDE:  VS Community 2017 15.7.3

**Comment by ajcvickers** [comment]
@natemcmaster Any insights on this?

**Comment by ajcvickers** [comment]
Sorry, commented too soon. @gnpretorius You will need to update to ASP.NET Core 2.1.1 as well, to keep all the versions in sync. This should be done by installing the new SDK, which will I believe will be available for download today.

**Comment by ralmsdeveloper** [comment]
@ajcvickers 
I had this problem, but it was because I had not installed the SDK.

https://github.com/dotnet/versions/tree/7a833dddfddc27f2074b755b94234a25b9757637/build-info/dotnet/product/cli/release/2.1

**Comment by gnpretorius** [comment]
Thanks guys!

I'll do that and follop up with the outcome.

**Comment by gnpretorius** [comment]
Hey guys. This is still not working for me? I've reinstalled the SDK (yesterday) and it's still version 2.1

Thoughts?

**Comment by ajcvickers** [comment]
@gnpretorius Please post your csproj files so we can see the dependencies.

**Comment by gnpretorius** [comment]
Here you go:
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Folder Include="wwwroot\" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>

</Project>
```

This is after I installed the latest SDK from here: 
https://www.microsoft.com/net/download/visual-studio-sdks

**Comment by gnpretorius** [comment]
Looks like this has finally been resoleased. Installed the 2.1.301 SDK and works fine now. 
https://blogs.msdn.microsoft.com/dotnet/2018/06/22/net-core-2-1-june-update/

Thanks 
Gordon

**Comment by natemcmaster** [comment]
Closing as it appears this is resolved and all 2.1.301 SDK downloads are available.
